Source/JavaScriptCore/wasm/WasmBBQPlan.cpp

```diff
@@ -60,6 +60,19 @@ BBQPlan::BBQPlan(Context* context, Ref<ModuleInformation> moduleInformation, uin
     dataLogLnIf(WasmBBQPlanInternal::verbose, "Starting BBQ plan for ", functionIndex);
 }
 
+bool BBQPlan::forceUsingB3(Ref<ModuleInformation> moduleInformation)
```

Comment:
Let's take `const ModuleInformation&`.

Reply:
I would suggest naming this function planGeneratesLoopOSREntrypoints